InjectionTDD/Classes/TDDTestKeeper.m

```diff
@@ -22,13 +22,56 @@ +(void)initialize{
     
     TDDInjectionTestsObserver *observer = [TDDInjectionTestsObserver new];
     [XCTestObservationCenter.sharedTestObservationCenter addTestObserver:observer];
-    
-    if (![[NSBundle bundleWithPath: @"/tmp/injectionforxcode/InjectionLoader.bundle"] load]) {
-        NSLog (@"Warning: Injection for Xcode not found. Please open Injection For Xcode.");
-    }else{
+
+    NSString *injectionForXcode = @"/tmp/injectionforxcode";
+    NSString *injectionIII = @"/Applications/InjectionIII.app/Contents/Resources/";
+
+
+#ifdef TARGET_OS_IOS
+    NSString *platformString = @"iOSInjection{version}.bundle";
+#endif
+
+#ifdef TARGET_OS_TV
+    platformString = @"tvOSInjection{version}.bundle";
```

Comment:
Is it compiling? Should it be `NSString *platformString = @"tvOSInjection{version}.bundle"`?

Reply:
From testing this, tvOS responds to both TARGET_OS_IOS and TARGET_OS_TV.
I'm gonna give it another try with a clean slate to see that it doesn't complain.
I actually tested the implementation with a tvOS application so... but then again, it could be the compiler cache playing tricks on me.

Reply:
I tried it again with the example project and one of my own tvOS projects and they both work as intended :)

Reply:
thanks for double-check!